ugandaemr-304.md

```diff
@@ -0,0 +1,58 @@
+## UgandaEMR 3.0.4 Release Notes
+
+Currently, work in progress
+
+### New Features
+1.  Patient Flags
+    * Due for third DNA PCR
+    * Over Due for third DNS PCR
+    * Updated Criteria for Due and Over Due for Second DNA PCR
+ 2. HIV MER Indicator Reports 
+    * TX_ML Report
+ 3. Data Extraction Reports
+    * Postnantal Register
+    
+    
+  
+### Fixes Made
+
+       1. Disabled the creation of Multiple encounters on the same visit date
+       2. Reports Module does not start even after starting the module
+       3. EID Card does not allow future dates for Next Appointment Date
+       4. System does not support to measure the number of patients seen by a perticular provider
```

Comment:
@solemabrothers Rewrite to Resolved issue with Data Entry Statistics that was ...